src/orchestrator/orchestrator.rs

```diff
@@ -646,19 +657,20 @@ impl<'a> JobTask<'a> {
         let dependency_receiving_span = tracing::debug_span!("receiving dependencies");
         while !all_dependencies_are_in(&self.jobdef.dependencies, &received_dependencies) {
             // Update the status bar message
-            self.bar.set_message({
-                format!(
-                    "[{} {} {}]: Waiting ({}/{})...",
-                    self.jobdef.job.uuid(),
-                    self.jobdef.job.package().name(),
-                    self.jobdef.job.package().version(),
-                    received_dependencies
-                        .iter()
-                        .filter(|(rd_uuid, _)| self.jobdef.dependencies.contains(rd_uuid))
-                        .count(),
-                    dep_len
-                )
-            });
+            self.bar.set_message(format!(
+                "{:-<MAX_ENDPOINT_NAME_LENGTH$} {:-<CONTAINER_ID_LENGTH$} {:-<JOB_UUID_LENGTH$} {} {} {} Waiting, ({}/{})",
+                "", // endpoint_name
+                "", // container_id
```

Comment:
Again :P

Edit: Quite a few more occurrences throughout the rest of the file :o :D